library/src/main/android/permissions/dispatcher/PermissionUtils.java

```diff
@@ -78,6 +78,25 @@ public static boolean hasSelfPermissions(Context context, String... permissions)
     }
 
     /**
+     * Determine context has access to the given permission.
+     *
+     * This is a workaround for RuntimeException of Parcel#readException.
+     * For more detail, check this issue https://github.com/hotchemi/PermissionsDispatcher/issues/107
+     *
+     * @param context context
+     * @param permission permission
+     * @return returns true if context has access to the given permission, false otherwise.
+     * @see #hasSelfPermissions(Context, String...)
+     */
+    private static boolean hasSelfPermission(Context context, String permission) {
+        try {
+            return checkSelfPermission(context, permission) == PackageManager.PERMISSION_GRANTED;
+        } catch (Throwable t) {
```

Comment:
Sure. Let me fix that. I was thinking just catch everything since this shouldn't happen.